tests/python/ci/test_ci.py

```diff
@@ -49,7 +38,7 @@ def run(pr_body, requested_reviewers, existing_review_users, expected_reviewers)
         git.run("remote", "add", "origin", "https://github.com/apache/tvm.git")
         reviews = [{"user": {"login": r}} for r in existing_review_users]
         requested_reviewers = [{"login": r} for r in requested_reviewers]
-        proc = subprocess.run(
+        proc = subprocess.run(  # pylint: disable=subprocess-run-check
```

Comment:
Looking at the `subprocess-run-check` linter rule, shouldn't we pass `check` explicitly to resolve it?

Reply:
It's checked manually to add some extra error info

Reply:
If you set `check=False`, that continues to work as far as I can tell?

Reply:
That still trips pylint as far as I can tell, it wants to see `check=True` only

Reply:
Can you check again, I tried it locally and it worked 🤔